pkgs/development/python-modules/tensorflow-addons/default.nix

```diff
@@ -0,0 +1,89 @@
+{ lib
+, buildBazelPackage
+, buildPythonPackage
+, fetchFromGitHub
+, python
+, rsync
+, setuptools
+, wheel
+, tensorflow
+, typeguard
+, pytestCheckHook
+}:
+
+let
+  pname = "tensorflow-addons";
+  version = "0.13.0";
+
+  bazel-wheel = buildBazelPackage {
+    name = "${pname}-${version}-py2.py3-none-any.whl";
+
+    src = fetchFromGitHub {
+      owner = "tensorflow";
+      repo = "addons";
+      rev = "v${version}";
+      sha256 = "1568cs2z6vdiidcddza443n5f8injv1z01an6adp1q68xqhs65pd";
+    };
+
+    nativeBuildInputs = [
+      python
+      rsync
+      setuptools
+      wheel
+    ];
+
+    # https://github.com/tensorflow/addons/blob/master/BUILD
+    bazelTarget = ":build_pip_pkg";
+
+    fetchAttrs = {
+      sha256 = "0sgxdlw5x3dydy53l10vbrj8smh78b7r1wff8jxcgp4w69mk8zfm";
+    };
+
+    TF_CXX11_ABI_FLAG = "0";
+    TF_HEADER_DIR = "${tensorflow.out}/lib/python3.8/site-packages/tensorflow/include/";
+    TF_SHARED_LIBRARY_DIR = "${tensorflow.out}/lib/python3.8/site-packages/tensorflow/";
+    TF_SHARED_LIBRARY_NAME = "libtensorflow_framework.so.2";
+
+    buildAttrs = {
+      preBuild = ''
+        patchShebangs .
```

Comment:
Can we make this more precise and through that faster?

Reply:
Seems like a side-quest until we can get the package to build.